sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/EpidemiologicalDataSteps.java

```diff
@@ -0,0 +1,517 @@
+            EpidemiologicalDataCasePage.WEARING_MASK_OPTIONS);
+    return YesNoUnknownOptions.valueOf(Text);
+  }
+
+  public YesNoUnknownOptions getWearingPPE() {
+
+    String Text =
+        webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+            EpidemiologicalDataCasePage.WEARING_PPE_OPTIONS);
+    return YesNoUnknownOptions.valueOf(Text);
+  }
+
+  public YesNoUnknownOptions getOtherProtectiveMeasuresc() {
+
+    String Text =
+        webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+            EpidemiologicalDataCasePage.OTHER_PROTECTIVE_MEASURES_OPTIONS);
+    return YesNoUnknownOptions.valueOf(Text);
+  }
+
+  public YesNoUnknownOptions getShortDistance() {
+
+    String Text =
+        webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+            EpidemiologicalDataCasePage.SHORT_DISTANCE_OPTIONS);
+    return YesNoUnknownOptions.valueOf(Text);
+  }
+
+  public YesNoUnknownOptions getFaceToFaceContact() {
+
+    String Text =
+        webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+            EpidemiologicalDataCasePage.LONG_FACE_TO_FACE_CONTACT_OPTIONS);
+    return YesNoUnknownOptions.valueOf(Text);
+  }
+
+  public YesNoUnknownOptions getAnimalMarket() {
+
+    String Text =
+        webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+            EpidemiologicalDataCasePage.ANIMAL_MARKET_OPTIONS);
+    return YesNoUnknownOptions.valueOf(Text);
+  }
+
+  public YesNoUnknownOptions getPercutaneous() {
+
+    String Text =
+        webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+            EpidemiologicalDataCasePage.PERCUTANEOUS_OPTIONS);
+    return YesNoUnknownOptions.valueOf(Text);
+  }
+
+  public YesNoUnknownOptions getContactToBlood() {
+
+    String Text =
+        webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+            EpidemiologicalDataCasePage.CONTACT_TO_BODY_FLUIDS_OPTONS);
+    return YesNoUnknownOptions.valueOf(Text);
+  }
+
+  public YesNoUnknownOptions getHandlingSamples() {
+
+    String Text =
+        webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+            EpidemiologicalDataCasePage.HANDLING_SAMPLES_OPTIONS);
+    return YesNoUnknownOptions.valueOf(Text);
+  }
+
+  public TypeOfPlace getTypeOfPlace() {
+    String Text =
+        webDriverHelpers.getValueFromCombobox(EpidemiologicalDataCasePage.TYPE_OF_PLACE_COMBOBOX);
+    return TypeOfPlace.fromString(Text);
+  }
+
+  public String getContinent() {
+    String Text =
+        webDriverHelpers.getValueFromCombobox(EpidemiologicalDataCasePage.CONTINENT_COMBOBOX);
+    return Text;
+  }
+
+  public String getSubContinent() {
+    String Text =
+        webDriverHelpers.getValueFromCombobox(EpidemiologicalDataCasePage.SUBCONTINENT_COMBOBOX);
+    return Text;
+  }
+
+  public String getCountry() {
+    String Text =
+        webDriverHelpers.getValueFromCombobox(EpidemiologicalDataCasePage.COUNTRY_COMBOBOX);
+    return Text;
+  }
```

Comment:
same as above, remove all get methods and put the lines straight where they are used